plottable.js

```diff
@@ -6106,6 +6106,9 @@ var Plottable;
                 if (rowPadding == null) {
                     return this._rowPadding;
                 }
+                if (rowPadding === Infinity || rowPadding < 0) {
+                    throw Error("rowPadding must be a positive finite value");
```

Comment:
nitpicking, but 0 is a valid input, so `non-negative` is a better description than `positive`